packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -94,6 +94,11 @@ class RunCommand extends RunCommandBase {
     argParser.addOption('use-application-binary',
         hide: !verboseHelp,
         help: 'Specify a pre-built application binary to use when running.');
+    argParser.addOption('kernel',
+        hide: !verboseHelp,
+        help: 'Path to a pre-built kernel blob to use when running.\n'
+              'This option only exists for testing new kernel code execution on devices\n'
+              'and is not needed during normal application development.');
```

Comment:
I would change "for testing new kernel code execution" to "for testing a new Dart VM execution model" or some such, just in case someone doesn't know what "kernel code" is. I don't want to start fielding bugs from enthusiastic users who think it's a Linux kernel or something.